AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -125,10 +126,15 @@ - (void)applyEnabledState:(BOOL)isEnabled {
     [MSAppDelegateForwarder addDelegate:self.appDelegate];
 
     // Enable the distribute info tracker.
-    NSString *distributionGroupId = [MS_USER_DEFAULTS objectForKey:kMSDistributionGroupIdKey];
-    [self.distributeInfoTracker updateDistributionGroupId:distributionGroupId];
     [self.channelGroup addDelegate:self.distributeInfoTracker];
-  } else {
+
+    // Update distribute info tracker with stored distributionGroupId
+    NSString *distributionGroupId = [MS_USER_DEFAULTS objectForKey:kMSDistributionGroupIdKey];
+    if (distributionGroupId) {
```

Comment:
Please reformat this if statement. Hard to read it.

Reply:
I agree.